firmware/targets/f7/stm32wb55xx_ram_fw.ld

```diff
@@ -49,14 +49,14 @@ ENTRY(Reset_Handler)
 /* Highest address of the user mode stack */
 _estack = 0x20030000;    /* end of RAM */
 /* Generate a link error if heap and stack don't fit into RAM */
-_Min_Heap_Size = 0x200;      /* required amount of heap  */
-_Min_Stack_Size = 0x400; /* required amount of stack */
+_Min_Heap_Size = 0x400;      /* required amount of heap  */
+_Min_Stack_Size = 0x1000; /* required amount of stack */
 
 /* Specify the memory areas */
 MEMORY
 {
-FLASH (rx)                 : ORIGIN = 0x08000000, LENGTH = 32K
-RAM1 (xrw)                 : ORIGIN = 0x20000008, LENGTH = 0x2FFF8
```

Comment:
0x20000008 I think I saw ERRATA or something like that. check required

Reply:
first bytes contain 2nd core state; this offset is needed so firmware doesn't overwrite them. If we don't use them, it's okay to start from 0x20000000

Reply:
is it guaranteed that core2 is not going to overwrite out memory?

Reply:
They are reserved for saving & reading OTA status on startup. See `P-NUCLEO-WB55.Nucleo\Applications\BLE\BLE_Ota\Core\Inc\app_conf.h`,  `CFG_OTA_REBOOT_VAL_MSG`
 ```
* Define list of reboot reason
 */
#define CFG_REBOOT_ON_FW_APP          (0x00)
#define CFG_REBOOT_ON_BLE_OTA_APP     (0x01)
#define CFG_REBOOT_ON_CPU2_UPGRADE    (0x02)

/**
 * Define mapping of OTA messages in SRAM
 */
#ifdef OTA_SBSFU
#define CFG_OTA_REBOOT_VAL_MSG            (*(uint8_t*)(LOADER_COM_REGION_RAM_START+4))
#define CFG_OTA_START_SECTOR_IDX_VAL_MSG  (*(uint8_t*)(LOADER_COM_REGION_RAM_START+5))
#define CFG_OTA_NBR_OF_SECTOR_VAL_MSG     (*(uint8_t*)(LOADER_COM_REGION_RAM_START+6))
#else
#define CFG_OTA_REBOOT_VAL_MSG            (*(uint8_t*)(SRAM1_BASE+0))
#define CFG_OTA_START_SECTOR_IDX_VAL_MSG  (*(uint8_t*)(SRAM1_BASE+1))
#define CFG_OTA_NBR_OF_SECTOR_VAL_MSG     (*(uint8_t*)(SRAM1_BASE+2))
#endif /* OTA_SBSFU */
```

Reply:
also minimal VTOR relocation is 0x200 IIRC, but if we can skip it, why bother

Reply:
So, to my understanding, that region is used as a semi-persistent area to store OTA state while restarting the MCU. If we don't follow the same practice, we can use it for our purposes.